index.js

```diff
@@ -579,19 +579,20 @@ const compile = function(schema, cache, root, reporter, opts) {
       consume('oneOf')
     }
 
-    if (node.multipleOf !== undefined) {
-      if (!Number.isFinite(node.multipleOf)) throw new Error('Invalid multipleOf')
+    const multipleOf = node.multipleOf !== undefined ? 'multipleOf' : 'divisibleBy' // draft3 support
```

Comment:
Nit: double negatives are confusing
```suggestion
    const multipleOf = node.multipleOf === undefined ? 'divisibleBy' : 'multipleOf' // draft3 support
```

Reply:
Fixed, thanks!

Reply:
Oh, this was a suggestion -- missed that, sorry ).